parent/pom.xml

```diff
@@ -179,6 +179,12 @@
                 <version>${org.springframework.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
What is that needed for? Can the slf4j bridge used in tests (assuming that's needed for Spring)?